src/node_v8.cc

```diff
@@ -295,6 +297,164 @@ static void SetHeapStatistics(JSONWriter* writer, Isolate* isolate) {
   writer->json_arrayend();
 }
 
+static Local<Object> ConvertHeapStatsToJSObject(
```

Comment:
For this method... for better error handling here, this should likely return `MaybeLocal<Object>` and the body should avoid making use of `ToLocalChecked(...)` and instead use `ToLocal(&...)`, returning `MaybeLocal<Object>()` when the `ToLocal(&...)` calls return false. I know that the `FIXED_ONE_BYTE_STRING(...)` calls still use `ToLocalChecked()` internally so this wouldn't completely handle every error condition correctly but it's a step in the right direction and I plan to fix up the `FIXED_ONE_BYTE_STRING(...)` uses fairly soon to improve those. There are plenty of examples in the codebase that have functions returning `MaybeLocal<...>` if you need some inspiration on how to get there.